src/v/cloud_storage/access_time_tracker.cc

```diff
@@ -21,37 +21,6 @@
 #include <exception>
 #include <variant>
 
-namespace absl {
-
-template<class Key, class Value>
-void write(iobuf& out, const btree_map<Key, Value>& btree) {
-    using serde::write;
-    write(out, static_cast<uint64_t>(btree.size()));
-    for (auto it : btree) {
-        write(out, it.first);
-        write(out, it.second);
-    }
-}
-
-template<class Key, class Value>
-void read_nested(
-  iobuf_parser& in,
-  btree_map<Key, Value>& btree,
-  size_t const bytes_left_limit) {
-    using serde::read_nested;
-    uint64_t sz;
-    read_nested(in, sz, bytes_left_limit);
-    for (auto i = 0UL; i < sz; i++) {
```

Comment:
i'm pretty sure we use 32 bit integer in serde to record the number of items in the map. Will this cause any issue with deployed code?